### PR TITLE
Remove duplicate PII from Client table

### PR DESCRIPTION
**Pull request by bengolder**

### CODE REVIEW

File: app/controllers/case_management/clients_controller.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def create
       # Don't create additional clients if we already have one
       return redirect_to case_management_client_path(id: intake.client_id) if intake.client_id.present?
 
-      client = Client.create_from_intake(intake)
+      client = Client.create!
```
Comment (bytheway875) [inline app/controllers/case_management/clients_controller.rb:21]: could also do Client.create!(intake: intake) which will associate the intake object to the client on creation, and remove line 22.

Comment (bytheway875) [reply app/controllers/case_management/clients_controller.rb:21]: Also, if we are leaving vita_partner_id on client, we def should create the client with the vita_partner_id of the intake!

```Client.create!(intake: intake, vita_partner_id: intake.vita_partner_id)```

**Review by bytheway875 — CHANGES_REQUESTED** [review]
We need to set vita_partner_id onto the client, I think!

File: app/controllers/case_management/clients_controller.rb (hunk 1 above)
Comment (sniemeyer13) [reply app/controllers/case_management/clients_controller.rb:21]: Thanks! we did exactly as you suggested, good catch

**Comment by sniemeyer13** [comment]
Incorporated comments and added a commit directly to master: https://github.com/codeforamerica/vita-min/commit/b063602f6dc897973390012f965b60b21245dac5

**Pull request closed without merging**